testfixture/testfixture.go

```diff
@@ -0,0 +1,33 @@
+package testfixture
```

Comment:
this name is too generic, try thinking of a name that better describes what this does.